include/psa/crypto_struct.h

```diff
@@ -326,9 +326,17 @@ typedef struct
 
 #define PSA_CORE_KEY_ATTRIBUTES_INIT {PSA_KEY_TYPE_NONE, 0, PSA_KEY_LIFETIME_VOLATILE, MBEDTLS_SVC_KEY_ID_INIT, PSA_KEY_POLICY_INIT, 0}
 
+
+#ifndef ALLOW_PRIVATE_ACCESS
+#define PSA_PRIVATE(member) private_##member
```

Comment:
We'd just use `MBEDTLS_PRIVATE` everywhere.

Reply:
replaced PSA_PRIVATE with MBEDTLS_PRIVATE